app/src/androidTest/java/com/sdp/movemeet/view/map/MainMapFragmentTest.java

```diff
@@ -165,7 +164,7 @@ public void cameraZoomsOnlyOnFirstCall() {
     public void clickingOnActivityMakesCorrectIntentWhenLogged() {
         MainMapFragment mapFragment = fragmentTestRule.getFragment();
         Activity act = new Activity("activity id", "organizer id", "my title", 4, new ArrayList<>(), 0, 0,
-                    "desc", null, new Date(), 1, Soccer, "Dubai UAE", new Date());
+                    "desc", "activities/test", new Date(), 1, Soccer, "Dubai UAE", new Date());
```

Comment:
Even if that's cleaner to set the documentPath of the activity this way, it was probably not necessary since this activity won't be actually uploaded to Firebase Firestore! But anyway, that's a good reflex, and it might help us figure out from where does our activity with a `null` documentPath get created at some point!

Reply:
True. I changed it because I thought the null documentPath bug in our db came from that, but as you stated, this activity isn't even uploaded to Firebase.